ck.sh

```diff
@@ -85,6 +85,53 @@ case "$COMMAND" in
 		esac
 		;;
 
+	buildskin)
+		cd skins
+
+		echo "Which skin do you want to build?"
+
+		select skin in *
+		do
+			echo "you selected $skin"
+			
+			cd ../ckeditor-dev
+
+			java -jar dev/builder/ckbuilder/2.3.2/ckbuilder.jar --build-skin ../skins/$skin ../ckeditor/skins/$skin
+
+			break
+		done
+		;;
+
+	createskin)
+		read -r -p "What is the name of the new skin? " skinName
+
+		echo "Which skin do you want to use as base?"
+
+		cd ckeditor-dev/skins
+
+		select opt in *
+		do
+			cd ../..
+
+			# Create skins folder if doesn't exist
+			mkdir -p skins
+
+			# Create folder for the new skin
+			mkdir skins/"$skinName"
```

Comment:
This is in part what `-p` is for. Rather than invoking `mkdir` twice:

```
     -p      Create intermediate directories as required.  If this option is not specified,
             the full path prefix of each operand must already exist.  On the other hand,
             with this option specified, no error will be reported if a directory given as an
             operand already exists.  Intermediate directories are created with permission
             bits of rwxrwxrwx (0777) as modified by the current umask, plus write and search
             permission for the owner.
```